src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -211,8 +212,8 @@ protected void describe(IceUdpTransportPacketExtension pe)
             fingerprintPE = new DtlsFingerprintPacketExtension();
             pe.addChildExtension(fingerprintPE);
         }
-        fingerprintPE.setFingerprint(dtlsStack.getLocalFingerprint());
-        fingerprintPE.setHash(dtlsStack.getLocalFingerprintHashFunction());
+        fingerprintPE.setFingerprint(dtlsStack.getCertificateInfo().getLocalFingerprint());
```

Comment:
non blocking but, would it make sense to store the certificate info in a local var to avoid the admittedly  improbable case where a different certificate info is returned?

Reply:
The cert in a DtlsStack instance never changes (it's a Kotlin 'val'), that's part of what the JMT PR did.